src/fides/api/models/sql_models.py

```diff
@@ -321,22 +335,54 @@ class System(Base, FidesBase):
     meta = Column(JSON)
     fidesctl_meta = Column(JSON)
     system_type = Column(String)
-    data_responsibility_title = Column(String)
-    joint_controller = Column(PGEncryptedString, nullable=True)
-    third_country_transfers = Column(ARRAY(String))
+    joint_controller = Column(
+        PGEncryptedString, nullable=True
+    )  # Will be deprecated in favor of System.joint_controller_info
```

Comment:
Is there a meaningful distinction between "will be deprecated" and "deprecated"?

Reply:
I'll change to deprecated, thanks